zh_cn/news/_posts/2018-03-28-unintentional-file-and-directory-creation-with-directory-traversal-cve-2018-6914.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: news_post
+title: "CVE-2018-6914: tempfile 和 tmpdir 库中意外创建文件和目录的缺陷"
+author: "usa"
+translator: "Delton Ding"
+date: 2018-03-28 14:00:00 +0000
+tags: security
+lang: zh_cn
+---
+
+Ruby 自带的 tmpdir 库中存在一个创建意外目录的缺陷。此外 tempfile 库也存在创建意外文件的缺陷，因为其内部依赖了 tmpdir。此缺陷已被分配 CVE 标识符  [CVE-2018-6914](http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-6914)。
```

Comment:
There is two space before [CVE-2018-6914].

Reply:
👌Fixed. Thank you.